src/upgrade/receiver.rs

```diff
@@ -8,20 +7,16 @@ use std::task::{Context, Poll};
 use crate::upgrade::Connection;
 
 /// The receiving half of a channel to send an upgraded connection.
-///
-/// Unlike `async_std::sync::channel` the `send` method on this type can only be
-/// called once, and cannot be cloned. That's because only a single instance of
-/// `Connection` should be created.
 #[must_use = "Futures do nothing unless polled or .awaited"]
```

Comment:
Is this comment no longer relevant? Why was it removed?

Reply:
This is the receiver, the comment was meant for the sender I assume.